### DISCUSSION TITLE
Support needed?

### DISCUSSION THREAD

**Discussion by Alox-K** [kingsleyadam/local-abbfreeathome-hass#115] [General]
Hi @kingsleyadam,
this project sounds very promissing. I made a quick installation and got it working immediately.
Since I do have a larger f@h installation with several devices, I'd like to offer my support to get more devices working. How can I help you?
Woudl you be interested in some kind of a video tutorial? On my german YT-channel, I have several f@h Videos... https://www.youtube.com/channel/UCxRxNlQIDkJ8KOZmNhLQXWg/
Would appreciate your answer.

**Comment by kingsleyadam** [comment]
Hey @Alox-K, since it sounds like you’ve got a pretty big free@home installation, were there any devices/entities missing from the integration that you’d expect? Or did everything come through that you’d expect?

If there’s anything missing maybe you can provide a diagnostics download from the integration and point out the missing devices?

**Reply by Alox-K** [reply]
Hey @kingsleyadam, yes, there were several devices missing, e.g. the new flex acutators, wheaterstation, roomtouch 4", on the dali gateway I'm missing the rgbw-led-strip, wireless door sensors,...

However, I'd love to help to improve this promissing project. I will try to check how to provide the diagnostics.

**Comment by Alox-K** [comment]
[config_entry-abbfreeathome_ci-01JE24484XGQBKFGC80RSWGH57.json](https://github.com/user-attachments/files/17998237/config_entry-abbfreeathome_ci-01JE24484XGQBKFGC80RSWGH57.json)
Here you are! Is this correct?

**Reply by kingsleyadam** [reply]
Yep, that's the one. If you could search for and find the missing devices in the file we can see how they can be implemented.

Usually easiest just to search by name you called it in Free@Home.

**Reply by derjoerg** [reply]
> [config_entry-abbfreeathome_ci-01JE24484XGQBKFGC80RSWGH57.json](https://github.com/user-attachments/files/17998237/config_entry-abbfreeathome_ci-01JE24484XGQBKFGC80RSWGH57.json) Here you are! Is this correct?

Hi @Alox-K,
first of all: I already seen several of your posts in the busch-jaeger community. Really, really great!

Your diagnostic is really big :smiley:

To narrow down this immensive information, can you point out the serialnumbers of the devices (e.g. ABB700DA8A77) you want to have included?

Just as an information, I noticed that you have an huge amount of scenes. Scenes are - as of now - not supported through the local REST API of F@H.

Joerg

**Reply by Alox-K** [reply]
>  first of all: I already seen several of your posts in the busch-jaeger community. Really, really great!

Thanks for your warm words! Compared to your affords about writing a whole plugin, my videos seem to be piece of cake :-)

**Comment by Alox-K** [comment]
Ok, i will try to narrow down the missing devices:

- On the DALI gateway ABB7026310B7 there is a fifth device, a led controller with cold and warm white (Schattenfuge Büro)
- 0007EE952146 is a Trevion Display with RTR (named Zantrale Schlafzimmer)
- ABB702FF8E80 (Treppe runter) is a switch from the Flextronics series with one channel, same as ABB702FF8E8D (Tür Bastelzimmer)
- Zentrale Büro (0007EE952B0E) is a Roomtouch 4", pretty similar to the Trevion Display. Contains a RTR
- ABB7F462BD1C (Küche Nische) is a little weird. Its not recognised in your plugin, but the same kind of device does apprear, such as ABB7F59EE8BB "Zwischen Fenstern (u)". Same for "Flur Treppe runter" ABB7F59A8B29

The kind you are considering forced positions and all the otther attributes is really great and opens a broad varienty of integrations. Awesome work!

**Comment by Alox-K** [comment]
Hey! May i asked if you found this Information helpful and if you see a chance to add this devices?

**Reply by derjoerg** [reply]
Hi,

sorry for the long delay. December is always busy with my "real" work 😆 

Looking at your list there are some low hanging fruits, which should be pretty easy to integrate:
- Trevion Display
- Flextronic switch
- Roomtouch
I will look at them, but I think we just need to update the supported FIDs of the existing devices.

The DALI gateway might be a bit more work as neither myself nor @kingsleyadam have such a device for testing (at least I think so), but together we will get it.

Regarding your last item, we need to check.

All of this stuff first needs to be supported by the underlying library https://github.com/kingsleyadam/local-abbfreeathome, so I will first open dedicated issues over there and then we can move over to the HA-integration.

**Reply by kingsleyadam** [reply]
On the `DALI Gateway` is looks like the channel in question is `FID_COLORTEMPERATURE_ACTUATOR`? Looks like this performs very similar to the `DimmingActuator` class but with some additional inputs/outputs to control the color `AL_INFO_COLOR_TEMPERATURE` output and `AL_COLOR_TEMPERATURE` input.

**Reply by kingsleyadam** [reply]
Regarding `0007EE952B0E`, this only has 1 function, `FID_SCENE_SENSOR` which isn't supported in the local api. At least I haven't seen any scene events come over the websocket with usable information. 

The other one referenced `ABB7F59EE8BB` is a `FID_DIMMING_SENSOR` which we do implement in the library, makes sense why that'd show up. 

The device `ABB7F59A8B29` is a `FID_STAIRCASE_LIGHT_SENSOR` which we don't implement and it wouldn't show up in home assistant. Although this is a pretty simple device that can probably be implemented, seems very similar to `DesDoorRingingSensor`

**Reply by derjoerg** [reply]
@Alox-K 
Without wanting to cause stress: Have you seen the question within https://github.com/kingsleyadam/local-abbfreeathome
